manifests/charts/istiod-remote/NOTES.txt

```diff
@@ -0,0 +1,3 @@
+Minimal istiod for Istio remote data plane. Mutating webhook configuration and sidecar injector configuration are included.
```

Comment:
`Minimal Istio deployment for remote data plane including a mutating webhook configuration and sidecar injection configuration.`

Reply:
updated